frontend/packages/dev-console/src/utils/useAddActionExtensions.ts

```diff
@@ -32,3 +40,68 @@ export const useAddActionExtensions = (): [ResolvedExtension<AddAction>[], boole
 
   return [allAddActionExtensions, resolved, allAddActionsDisabled];
 };
+
+export const getEnabledSubCatalogTypes = (catalogType: string): boolean => {
```

Comment:
```suggestion
export const isSubcatalogTypeEnabled = (catalogType: string): boolean => {
```

Reply:
I would recommend `isCatalogTypeEnabled` as the extension is called `CatalogType`